### PR TITLE
Fix for ESP8266 sample

### PR DESCRIPTION
**Pull request by ewertons**
- Separate the root CA header generation from the script that creates the
Arduino package;
- Create similar script to generate CA cert header on Windows Powershell
- Update Quick Guide to reflect the additional steps

### CODE REVIEW

**Review by CIPop — APPROVED** [review]
Thanks @ewertons !

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 1]
```diff
@@ -109,7 +110,31 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
 5. Create a sketch on Arduino IDE for the IoT Hub telemetry sample.
 
-    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and then open the [ESP8266 sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/aziot_esp8266) (from the local clone) on the Arduino IDE.
+    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and
```
Comment (ericwol-msft) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:113]:
```suggestion
    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally
```

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 2]
```diff
@@ -109,7 +110,31 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
 
 5. Create a sketch on Arduino IDE for the IoT Hub telemetry sample.
 
-    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and then open the [ESP8266 sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/aziot_esp8266) (from the local clone) on the Arduino IDE.
+    - Clone the [Azure SDK for Embedded C](https://github.com/Azure/azure-sdk-for-c) repository locally and 
+
+    - Generate the `ca.h` header (in the ESP8266 sample folder!) with the public root CA for server certificate validation
+
+      - Navigate to the ESP8266 sample in your local clone
```
Comment (ericwol-msft) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:117]:
```suggestion
      - Navigate to the ESP8266 sample in your local cloned repo
```

**Review by ericwol-msft — APPROVED** [review]

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 3]
```diff
@@ -109,7 +110,31 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
...
+
+        ```bash
+        cd <cloned repo root>/sdk/samples/iot/aziot_esp8266
+        ```
+
+      - Run the script to generate the `ca.h` header
+
```
Comment (momuno) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:124]: Nit: Add a '.' to the end of instruction sentences.

**Comment by momuno** [comment]
This is great.  Thank you @ewertons for updating the scripts. I'm linking the reported issue to this so it gets closed automatically when you merge.

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md (hunk 3 above)
Comment (ewertons) [reply sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:124]: Fixed.

**Pull request merged**